docs/en/latest/concepts/annotations.md

```diff
@@ -204,3 +204,58 @@ spec:
             port:
               number: 80
 ```
+
+Use ApisixPluginConfig
+---------
+
+You can use the following annotations to use the `ApisixPluginConfig`.
+
+* `k8s.apisix.apache.org/plugin-conifg-name`
+  
+If this annotations set to `ApisixPluginConfig.metadata.name` the route will use `ApisixPluginConfig`
+
+If this comment is set to the `ApisixPluginConfig` resource name, the route will set the corresponding `PluginConfig`
+
+ApisixPluginConfig is a resource under the same Namespace as Ingress
```

Comment:
@juzhiyuan @navendu-pottekkat  Can you guys help improve this documentation?

Reply:
If the content is correct, you can merge first. Navendu will update this project's docs when available.

Reply:
okay